build.rs

```diff
@@ -1,18 +1,17 @@
-const BUILD_DIR: &str = "src/gen";
```

Comment:
You should remove code from this file roo

Reply:
done: https://github.com/bipa-app/lnd-rs/pull/65/commits/4f2c308ca94ad947663de6eee0cfa4621564ba24